cli/src/template.rs

```diff
@@ -201,7 +201,7 @@ describe("{}", () => {{
   it("Is initialized!", async () => {{
     // Add your test here.
     const program = anchor.workspace.{};
-    const tx = await program.rpc.initialize();
+    const tx = await program.methods.initialize();
```

Comment:
Need to change to `program.methods.initialize().rpc()`

Reply:
Done!